src/openloco/config.h

```diff
@@ -9,10 +9,13 @@ namespace openloco::config
 
     struct config_t
     {
-        uint32_t unk_00;
+        uint8_t show_height_as_units;
+        uint8_t pad_01[3];
         uint16_t resolution_width;
         uint16_t resolution_height;
-        uint8_t pad_06[526];
+        uint8_t pad_04[32];
+        uint8_t measurement_format;
+        uint8_t pad_06[493];
```

Comment:
Could someone verify I have implemented these correctly? For reference: both of the implemented vars are used around `0x495A2F`.

Reply:
That is not correct. The existing structure is also not correct. Resolution height is at offset 6. Padding starts at offset 8. There is padding to take it to 0x24 for measurement format. We also have stuck to a different way of writing padding in other structures. It should be offset of next item minus current offset to get the size. See the vehicle thing structure

Reply:
Okay, I have adjusted the labels. Could you look it over once more?

Reply:
```
uint8_t show_height_as_units; // 0x00
uint8_t pad_01[3];
uint16_t resolution_width; // 0x04
uint16_t resolution_height; // 0x06
uint8_t pad_04[32]; // <- make this pad_08[0x24 - 0x08]
uint8_t measurement_format; // 0x24
uint8_t pad_06[493]; // <- make this pad_25[SIZE - 0x25]
```

Reply:
Thanks @marijnvdwerf. Unfortunately, you've commented on an outdated version of the struct. I'd already [changed](https://github.com/OpenRCT2/OpenLoco/pull/154/commits/845ded4376d089b37a43d5a747b60f508018cd4e) things little based on Duncan's suggestions. Do you agree with those changes, too?